spec/system/kudo/create_spec.rb

```diff
@@ -51,14 +56,47 @@
       it 'shows current form with entered data and displays error' do
         visit new_kudo_path
         fill_in 'kudo_title', with: 'Lorem ipsum dolor sit amet'
-        last_option = page.all('option').last.text
-        page.select last_option, from: 'kudo_reciever_id'
+        last_option_reciever = page.find_field('Reciever').all('option').last.text
+        page.select last_option_reciever, from: 'kudo_reciever_id'
+        last_option_company_value = page.find_field('Company value').all('option').last.text
+        page.select last_option_company_value, from: 'kudo_company_value_id'
         click_on 'Save Kudo'
         expect(body).to have_css "form[action='/kudos'][method='post']"
-        expect(body).to have_select 'kudo_reciever_id', selected: last_option
+        expect(body).to have_select 'kudo_reciever_id', selected: last_option_reciever
+        expect(body).to have_select 'kudo_company_value_id', selected: last_option_company_value
         expect(body).to have_field 'kudo_title', with: 'Lorem ipsum dolor sit amet'
         expect(body).to have_content "Content can't be blank"
       end
     end
+
+    context 'when missing reciever' do
```

Comment:
FYI - You do not really have to check every possible error here, sine you have covered it in the model specs already - one error should be enough to make sure, that notice is displayed